neuro_evolution_ctrnn/brain_visualizer/__init__.py

```diff
@@ -1,40 +1,260 @@
+                if event.type == pygame.KEYDOWN:
+                    if event.key == pygame.K_ESCAPE:
+                        pygame.quit()
+                        Positions.clearJSON(self)
+                        sys.exit()
+                    if event.key == pygame.K_e:
+                        self.weightVal = self.weightVal - 1
+                    if event.key == pygame.K_r:
+                        self.weightVal = self.weightVal + 1
+                    if event.key == pygame.K_d:
+                        if self.neuronRadius > 5:
+                            self.neuronRadius= self.neuronRadius - 5
+                            print(self.neuronRadius)
+                    if event.key == pygame.K_f:
+                        self.neuronRadius = self.neuronRadius + 5
+                    if event.key == pygame.K_s:
+                        if self.neuronText:
+                            self.neuronText = False
+                        else:
+                            self.neuronText = True
+                    if event.key == pygame.K_t:
+                        if self.positiveWeights:
+                            self.positiveWeights = False
+                        else:
+                            self.positiveWeights = True
+                    if event.key == pygame.K_w:
+                        if self.negativeWeights:
+                            self.negativeWeights = False
+                        else:
+                            self.negativeWeights = True
+                    if event.key == pygame.K_g:
+                        if self.weightsDirection:
+                            self.weightsDirection = False
+                        else:
+                            self.weightsDirection = True
+                    if event.key == pygame.K_q:
+                        if self.inputWeights:
+                            self.inputWeights = False
+                        else:
+                            self.inputWeights = True
+                    if event.key == pygame.K_z:
+                        if self.outputWeights:
+                            self.outputWeights = False
+                        else:
+                            self.outputWeights = True
+                    if event.key == pygame.K_SPACE:
+                        pause = True
+                        pygame.event.clear(KEYDOWN)
+                        while pause:
+                            for event in pygame.event.get():
+                                if event.type == pygame.KEYDOWN:
+                                    if event.key == pygame.K_c:
+                                        pause = False
+
+            except AttributeError:
+                print("Failure on Pygame-Event")
+                continue
+
+        # Updates the content of the window
+        pygame.display.flip()
+
+    def render_InfoText(self, list_of_strings, x_pos, initial_y_pos, color, y_step):
```

Comment:
```suggestion
    def render_InfoText(self, list_of_strings, x_pos, initial_y_pos, y_step):
```
`color` wird hier nicht genutzt